vendor/github.com/kubernetes-csi/external-snapshotter/client/v4/apis/volumesnapshot/v1/types.go

```diff
@@ -179,7 +179,7 @@ type VolumeSnapshotStatus struct {
 	// This field could be helpful to upper level controllers(i.e., application controller)
 	// to decide whether they should continue on waiting for the snapshot to be created
 	// based on the type of error reported.
-	// The snapshot controller will keep retrying when an error occurrs during the
```

Comment:
You don't have to fix the typos in the `vendor` directory, those are typos made by our dependencies, not us, so the test is failing because it thinks the PR is changing dependency code when we want the dependencies to be frozen.